### ISSUE TITLE
Question: Editing sprites with Pinta

### ISSUE THREAD

**Issue by joeskeen** [PintaProject/Pinta#1202]
A sprite sheet is a single image which is subdivided into columns/rows containing frames for the animation of game objects.

Obviously, I can open a sprite sheet (.PNG) file in Pinta, and make changes; however, it would be really nice to be able to have each frame of a sprite shown in its own layer, and position each on top of the other. This would make it a lot easier to fix issues like alignment as you could make a reference frame and a problem frame visible, overlaid on each other, and be able to move the pixels on the one frame to line up better with the reference.

Now I know that Pinta isn't primarily a sprite editor, but I've been trying to find a way to do this and not finding a lot of good options. Anyone aware of either a good program that would allow this kind of editing to be possible, or would it be possible to create an add-on for Pinta to be able to do this?

**Comment by cameronwhite** [comment]
Yeah there isn't a good way to do this in Pinta currently, but I think it would be possible for an add-on to either load an image directly in that format, or take the existing open image and convert to layered tiles.

I'm not familiar enough with sprite editing to have suggestions for other program though :)
